schemas/telemetry/account-ecosystem/account-ecosystem.4.schema.json

```diff
@@ -1,5 +1,15 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
+  "mozPipelineMetadata": {
+    "bq_dataset": "account_ecosystem",
+    "bq_table": "desktop",
+    "jwe_mappings": [
+      {
+        "decrypted_field_path": "payload.ecosystemUserId",
+        "source_field_path": "payload.ecosystemAnonId"
+      }
+    ]
```

Comment:
Just as a note, some additional metadata we're planning on leveraging are the following:

* retention rules, both partition retention (standard case) and fixed date retention (special case for pioneer), the former of which we could apply to the activity-stream namespace
* data categorization, which depending on how this evolves may not be used directly but I still think is worth having in metadata

I have a vague notion that including pipeline family eventually in metadata would help me automate some of the ops provisioning logic, which likely would only matter in the case we start to branch out into multiple bq projects or separate pipeline families for high-category data.